tests/test_reacts/test_reacts_cz.py

```diff
@@ -105,9 +104,13 @@ def test_react_czech_regex_yes_match(test_str: str):
 
 @pytest.mark.parametrize(
     "test_str",
-    POSTITIVE_TEST_CASES,
+    POSITIVE_TEST_CASES,
 )
 def test_react_czech_regex_no_match(test_str: str):
     """Tests that these strings return FALSE."""
-    # * surrounded by text
-    assert not check_resource_match(f"a{test_str}b", resource_name='czech')
+    if test_str == "捷克":
```

Comment:
Conflicted on hardcoding the text string, not ideal imo 
IIRC the Ukrainian test has mandarin handling, check out how it's done in there instead